src/fusor/harvester.py

```diff
@@ -293,7 +293,9 @@ def __init__(
 
         civic.load_cache(local_cache_path=local_cache_path, on_stale="ignore")
         self.translator = CIVICTranslator(fusor=fusor)
-        self.fusions_list = None
+
+        # Load in accepted fusion variants from CIViC
+        self.fusions_list = civic.get_all_fusion_variants(include_status="accepted")
```

Comment:
This is going to cause unit-tests that instantiate a CIVICHarvester to load data from the public CIVIC servers. Should they be loading cached/fixture data instead of making network calls?

Reply:
@zealws So we actually use a smaller test cache file in the fixtures directory, so we're not loading the entire cache in the CIViC tests.

Reply:
Sorry, I'm a bit confused by the question? Are you describing the test cache (located in the `fixtures` directory), or the list of accepted fusion variants that are loaded from the user's civicpy cache?

Reply:
Correct, default allows cache:

https://github.com/griffithlab/civicpy/blob/328c7918e469e6f390775bfd0d466839fc873863/civicpy/civic.py#L2525-L2534

However, @jarbesfeld you should make sure to use the test cache path when instantiating `CIVICHarvester` in the tests rather than relying on the default path that civic provides.

Reply:
@korikuzma So the test cache is this case is not an actual civicpy cache, but a list of three fusion variants. The parameter in the harvester expects the path to the actual civicpy cache, so this wouldn't work here. Is there something different I should be doing?